package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "buying-catalogue-components",
-  "version": "1.2.5",
+  "version": "1.2.6",
```

Comment:
I think this should be a minor update if not major – would you consider this a breaking change? Will we need to make changes to the consuming applications once we update to this latest version of the package? If so, then major; otherwise, minor.

Reply:
erm we'll need to change the bc-table-flexbile components in the order form so would that mean a major change?

Reply:
Yes, I think so. Let's do a major version bump, to version 2.x.x then.

Reply:
2.0.1? or 2.0.0?

Reply:
2.1.0 or 2.0.0?

Reply:
Probably `2.0.0` but I think the `npm version major` command will handle that for us.

Reply:
yeah fair enough